exercises/B-boolean-literals/exercise.js

```diff
@@ -6,6 +6,12 @@
 */
 
 let codeYourFutureIsGreat = true;
+let mozafarIsCool = false;
+let calculationCorrect = true;
+let moreThan10Students = false;
```

Comment:
Try to use `const` instead of `let` - if the variable is never being reassigned it's best to use `const` as it informs other devs that the variable _shouldn't_ change